include/userobjects/ThresholdHeuristicUserObject.h

```diff
@@ -0,0 +1,30 @@
+#pragma once
+
+#include "ClusteringHeuristicUserObjectBase.h"
+
+/*A clustering heuristic user object which clusters two neighboring elements whose scores
```

Comment:
Check your Doxygen multiline comments, I don't think this one will get rendered correctly. They should begin with `/**` and end with `*/`